src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -86,8 +98,14 @@ const ProgressTracker = ({
             <StyledValue>
               {label(currentProgressLabel, `${progress}%`)}
             </StyledValue>
+            {/* <StyledValuesLabel>of</StyledValuesLabel> */}
+            {/* {maxProgressLabel === "100%" ? (
+              <StyledValuesLabel>of</StyledValuesLabel>
+            ) : (
+              " "
+            )} */}
             <StyledValue isMaxValue>
-              {label(maxProgressLabel, "100%")}
+              {label(maxProgressLabel, "100%" && description)}
```

Comment:
I appreciate you are still working on the labelling currently so this is subject to change, but in this example I would personally refactor the function so all the logic is handled inside it instead of passing a boolean expression like this. This helps makes the function easier to read and understand.

Reply:
This comment doesn't look like it's been addressed but the conversation has been resolved? I'm inclined to agree with James

Reply:
It hasn't been address as I am not sure what would be the best approach to this one....